### DISCUSSION TITLE
[WARNING PDN-0195] Removing 3 via(s) between M2 and M5 at (2211.3120 um, 482.4900 um) for VDD

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#4215] [General]
What is OpenROAD trying to tell me here? 221.3120, 482.4900 is at the "W" in the image below:

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/a30f8201-095a-4c31-9e75-04c320159f48)

**Comment by maliberty** [comment]
That it can't legally put any via here to make a connection between these shapes.  It isn't necessarily a problem but it could be so it lets you know.
